### DISCUSSION TITLE
Feature Request: Swap config files with external switch

### DISCUSSION THREAD

**Discussion by legodude** [ZuluSCSI/ZuluSCSI-firmware#863] [Ideas]
I would love to be able to swap configuration files based on an external switch. This would enable seamless dual booting on systems (ie, vintage Unix) which do not easily support two separate operating systems, and would have the added benefit that the two OS installs would be completely partitioned from each other.

**Comment by aperezbios** [comment]
@legodude without knowing which ZuluSCSI hardware model you were requesting this for, it's non-actionable on our end. I know this is old (562 days) but it would be helpful if you
